CONTRIBUTING.md

```diff
@@ -43,14 +49,50 @@ Suppose for your PR you create a branch on `root.git`.
 Our CI infrastructure automatically picks up a branch with the same name in your fork of `roottest.git`
 and use that for testing your PR.
 
+Building ROOT with your changes
+-----------------------
 
-Continuous Integration
-----------------------
+The source code for ROOT is kept in [GitHub](https://github.com/root-project/root).
+In general, with the sources in `../src`, you build ROOT with `cmake -Droottest=On -Dtesting=On -Dall=On ../src && cmake --build`.
+If possible, configure with ninja, passing `-G Ninja` to the first `cmake` invocation.
+If any of this doesn't work for some reason then you can find more details [here](https://root.cern/install/build_from_source/).
+
+Testing ROOT with your changes
+-----------------------
 
 To prevent bad surprises and make a better first impression, we
 strongly encourage new developers to [run the tests](https://root.cern/for_developers/run_the_tests/)
 _before_ submitting a pull request.
 
+To run ROOT's test suite you need to execute `ctest -j 12`. The number (12 in this example) depends on how many cores you have.
```

Comment:
I think that the testing is not on by default.  Should we mention here the required `-D` for the `CMake` invocation?

Reply:
It's [right above](https://github.com/root-project/root/pull/13676/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R56)?

```txt
In general, with the sources in `../src`, you build ROOT with `cmake -Droottest=On -Dtesting=On -Dall=On ../src && cmake --build`.
```

Reply:
yep but I did not see it :(